content/en/docs/developer-documentation/build-install-rpm.md

```diff
@@ -24,10 +24,11 @@ sudo dnf install -y git golang rpm-build selinux-policy-devel container-selinux
 
 Clone the repository and cd into it:
 {{< warning >}}
-Make sure to use the `4.8.0-microshift-2022-04-20-141053` branch when working with the source repository. 
-Later versions of the source code are not compatible with the community packages (to be fixed in the future).
-{{< /warning >}}
+Community documentation is not currently compatible with the latest source code (to be fixed in the future).
```

Comment:
Leaving a single review comment considering the text is repeated.

```markdown
The available community documentation is not currently compatible with the latest MicroShift source code.
To build the latest MicroShift RPMs, follow the instructions in the [openshift/microshift Git repository](https://github.com/openshift/microshift).
Otherwise, use the `4.8.0-microshift-2022-04-20-141053` branch when working with the source repository.
```

Reply:
I like the proposed change. You might make the last sentence "Otherwise use the ... branch when working with the source repository and these instructions."